kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/SessionManager.java

```diff
@@ -220,6 +231,15 @@ public void start() {
         }
     }
 
+    private boolean isRunningFromMaven() {
+        try {
+            SessionManager.class.getClassLoader().loadClass("org.apache.maven.surefire.booter.ForkedBooter");
```

Comment:
Does it also work when we don't fork surefire/failsafe test execution? See http://maven.apache.org/surefire/maven-surefire-plugin/examples/fork-options-and-parallel-execution.html for more details

Reply:
yes it works in both the cases